src/core/components/ConfigModel.tsx

```diff
@@ -144,18 +151,24 @@ export default function ConfigModel({ handleOpen, handleClose, handleParentModal
     }
 
     const handleValidation = async () => {
+       
+        const providerExists = customProviderList.some((provider: { providerId: string; }) => provider.providerId === providerId);
+
         setShowErrorAlert(true);
         if (!providerId) {
-            setAlertMsg('Provider Id')
-        } else if (!authorizationUrl) {
-            setAlertMsg('Authorization URL')
+            setAlertMsg('Provider Id is required')
```

Comment:
The messages should be part of I18N files right?

Reply:
Yes Vivek. 
I have added now